galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.cemt.manager/src/main/java/dev/galasa/cicsts/cemt/internal/CEMTManagerField.java

```diff
@@ -0,0 +1,12 @@
+package dev.galasa.cicsts.cemt.internal;
```

Comment:
This needs to be in the dev.galasa.cicsts.cemt package.   the older managers are wrong.   BND objects to it being in a private package